pkgs/development/tools/rust/cargo-valgrind/default.nix

```diff
@@ -8,18 +8,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-valgrind";
-  version = "1.3.0";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "jfrimmel";
     repo = "cargo-valgrind";
-    # repo lacks the 1.3.0 tag
+    # repo lacks the 2.0.0 tag
     # https://github.com/jfrimmel/cargo-valgrind/issues/33
```

Comment:
This issue and comment are no longer needed as the tag has been pushed.

Reply:
Please open a PR with this update and the fix.

Reply:
I thought it was going to be done by the PR proposer. I'll look at this tomorrow.

Reply:
@otavio r-ryantm is a bot that automatically makes PRs